tox.ini

```diff
@@ -14,7 +14,7 @@ commands =
 [testenv:pytest]
 # Run the python tests.
 # To execute, run `tox -e pytest`
-envlist = py37, py38, py39, py310, py311
+envlist = py37, py38, py39, py310, py311, py312
 commands =
     poetry install -v
     poetry run pytest
```

Comment:
While being at it: It feels like Tox isn't used much any longer these days. Let me know if you still use it, or if you think `tox.ini` should also be removed.

Reply:
Probably doesn't hurt to keep for now